src/pages/post/Post.vue.scss

```diff
@@ -63,9 +63,9 @@
     // make a class for every category name in $POST_CATEGORIES and set the correct background and color
     @each $category, $cat-color in $POST_CATEGORIES {
       &.#{$category} {
+        background-color: $cat-color;
 
         &:before {
```

Comment:
The` :before` is now no longer used, can just delete this section. Because the text container is always white, the lightness detection part can also be removed.